packages/stitch/tests/mergeDeepInterfaces.test.ts

```diff
@@ -108,29 +108,11 @@ const decorators = [
 
 const reviews = [{ id: 'tx1', productUpc: '1', userId: '1', body: 'love it' }];
 
-const localResolvers = {
+const localResolvers: IResolvers = {
   Transaction: {
-    decorators: (_, { ids }) => {
+    decorators: ({ id }: { id: string }) => {
```

Comment:
This was wrong because there is no argument `ids` but it needs `id` from `Transaction` to fetch `decorators`